src/typed-method-types/files.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseResponse } from "../types.ts";
+
+export type FileUploadV2 = {
+  /** @description Description of image for screen-reader. */
+  alt_text?: string;
+  /** @description Syntax type of the snippet being uploaded. */
+  snippet_type?: string;
+  /** @description The message text introducing the file in specified channels. */
+  channel_id?: string;
+  /** @description Provide another message's ts value to upload this file as a reply. Never use a reply's ts value; use its parent instead. */
+  thread_ts?: string;
+  /** @description The message text introducing the file in specified channels. */
+  initial_comment?: string;
+  /** @description Title of the file being uploaded */
+  title?: string;
+
+  /** @description Size in bytes of the file being uploaded. */
+  length: string;
+  /** @description Name of the file being uploaded. */
+  filename: string;
+  /** @description Filetype of the file being uploaded. */
+  file: Blob | ReadableStream<Uint8Array> | string | ArrayBuffer;
```

Comment:
I think the `content` field is missing from this type 🤔  ([node sdk content field](https://github.com/slackapi/node-slack-sdk/blob/4f393719e76236d8d3eb9e5961e6837045e445b4/packages/web-api/src/methods.ts#L1987)) was this omitted for a specific reason?

Reply:
I didn't implement the content field because it's not usable for file upload v2 (when uploading via getExternalUploadURL). In node-slack-sdk, whether you use the content or file field, the data is processed like the file field and an HTTP request is made. [reference](https://github.com/slackapi/node-slack-sdk/blob/4f393719e76236d8d3eb9e5961e6837045e445b4/packages/web-api/src/file-upload.ts#L108)

Reply:
I learned from @filmaj that the content field is an important element! I was mistaken in my previous comment, so I will implement the content field. 🙇‍♂️

https://github.com/slackapi/deno-slack-api/pull/90#discussion_r1430540638